.github/workflows/pre-release.yml

```diff
@@ -119,6 +119,9 @@ jobs:
       DISPLAY: ""
       CYPRESS_ALL_FEATURES_TOKEN: ${{ secrets.ENTERPRISE_TOKEN }}
       CYPRESS_NO_FEATURES_TOKEN: ${{ secrets.E2E_STARTER_TOKEN }}
+      # disabled because of out of memory issues
+      # probably related to https://github.com/cypress-io/cypress/issues/27415
+      CYPRESS_NO_COMMAND_LOG: 1
```

Comment:
If you really want to test this out, would it be better to test this in other workflows that are run more often e.g. `e2e-tests.yml` workflow? Since this workflow is only run when we release a new Metabase version.

Reply:
we can't run it on e2e-tests workflow as I understand because we use replay and afaik it uses this feature from cypress

Reply:
That makes sense

Reply:
You can (and should) test this in your Metabase fork.
The workflow is too important to mess with it, imo.

Reply:
@nemanjaglumac if it doesn't work, we can revert this change

Reply:
testing in my fork now
https://github.com/iethree/metabase/actions/runs/8740359888